anyblok/config.py

```diff
@@ -822,14 +822,6 @@ def add_doc(group):
                        help='No detail these models')
```

Comment:
@jssuzanne as I understand, we can't directly give the ``-c app.test.cfg`` file in command line using pytest. It would be probably nice to offer a default value populate from env varaibles likes this:

```python

@Configuration.add('config')
def add_configuration_file(parser):
    parser.add_argument('-c', dest='configfile',
                        default=os.environ.get('ANYBLOK_CONFIG_FILE'),
                        help="Relative path of the config file")
``` 

what do you think about ?

Reply:
You are write. I will do it.

Reply:
thanks